src/main/java/io/gatling/mojo/AbstractEnterprisePluginMojo.java

```diff
@@ -32,8 +32,11 @@
 
 public abstract class AbstractEnterprisePluginMojo extends AbstractEnterpriseMojo {
 
-  @Parameter(defaultValue = ConfigurationConstants.Url.DEFAULT, readonly = true)
-  protected URL enterpriseUrl;
+  @Parameter(defaultValue = ConfigurationConstants.ApiUrl.DEFAULT, readonly = true)
+  protected URL enterpriseApiUrl;
+
+  @Parameter(defaultValue = ConfigurationConstants.WebAppUrl.DEFAULT, readonly = true)
+  protected URL enterpriseWebAppUrl;
```

Comment:
Not backwards-compatible (changes the name used in the POM.xml file), but this setting was never documented for end users.